examples/Image-Data-Extraction-API.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
We banished notebooks a while ago as they don't play nice with git, CC @abhiskk

Reply:
I had asked Michael to make one so that it serves as a tutorial. What would be better if not a notebook?

Reply:
We banished them from git, they still exist, just not in git.

Reply:
Oleksandr said I should make a tutorial on the habitat website that reflects the same information